docs/content/querying/dimensionspecs.md

```diff
@@ -414,6 +414,23 @@ or without setting "locale" (in this case, the current value of the default loca
 }
 ```
 
+### Bucket Extraction Function
+
+Bucket extraction funciton is used to bucket numerical values in each range of the given size by converting them to the same base value. Non numeric values are converted to empty strings.
```

Comment:
> Non numeric values are converted to empty strings.

I would rather this said "Non numeric values are converted to null." this is the same if I understand since `"" == null` in Druid but to someone coming in "null" make a lot more sense than "empty string" in this context.